packages/kmod-6.1-nvidia/nvidia-open-gpu-modules.service.in

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Copy Open GPU drivers to needed modules directory
+Before=load-kernel-modules.service
+After=systemd-tmpfiles-setup.service
+DefaultDependencies=no
+RefuseManualStart=true
+RefuseManualStop=true
+ConditionPathExists=/run/nvidia/nvidia-open-gpu
+
+[Service]
+Type=oneshot
+# This ensures the directory is created before copying. The directory is created by tmpfiles but udev kicks this off early in boot
+ExecStart=/usr/bin/mkdir -p __NVIDIA_MODULES__
```

Comment:
I don't understand this comment. You won't run until after tmpfiles runs so why does it matter that udev is what activates the job?

Reply:
I've removed this from the approach and this should not need these `mkdir` calls.